src/guiFormSpecMenu.cpp

```diff
@@ -2442,6 +2433,12 @@ void GUIFormSpecMenu::drawMenu()
 		core::rect<s32> imgrect(0, 0, spec.geom.X, spec.geom.Y);
 		// Viewport rectangle on screen
 		core::rect<s32> rect = imgrect + spec.pos;
+		if (spec.parent_button && spec.parent_button->isPressed()) {
+			core::dimension2d<s32> dim = core::dimension2d<s32>(
+				skin->getSize(gui::EGDS_BUTTON_PRESSED_IMAGE_OFFSET_X),
+				skin->getSize(gui::EGDS_BUTTON_PRESSED_IMAGE_OFFSET_Y));
+			rect += dim;
```

Comment:
and do directly the += operation on the core::dimensiond2, don't assign a useless variable